manim/mobject/coordinate_systems.py

```diff
@@ -256,28 +265,86 @@ def create_axis(self, range_terms, axis_config, length):
             axis.shift(-axis.number_to_point(0))
         return axis
 
-    def coords_to_point(self, *coords):
+    def coords_to_point(self, *coords: Sequence[float]) -> np.ndarray:
+        """Returns back a point with respect to the origin from the given coordinates.
+
+        Returns
+        -------
+        np.ndarray
+            A point formed by the coordinates
+        """
         origin = self.x_axis.number_to_point(0)
         result = np.array(origin)
         for axis, coord in zip(self.get_axes(), coords):
             result += axis.number_to_point(coord) - origin
         return result
 
-    def point_to_coords(self, point):
+    def point_to_coords(self, point: float) -> Tuple:
+        """Returns back the coordinates of a particular point.
```

Comment:
Same as above, I don't think `ccordinates of a particular point` is descriptive enough. It transforms the coordinates of the `point` which are with respect to manim's default basis into the coordinates of that `point` with respect to the basis defined by `Axes`. (inverse operation of `coords_to_point`)